py3status/modules/taskwarrior.py

```diff
@@ -38,7 +41,11 @@ def taskWarrior(self):
         def describeTask(taskObj):
             return str(taskObj['id']) + ' ' + taskObj['description']
 
-        task_command = 'task start.before:tomorrow status:pending export'
+        if self.filter:
+            task_command = 'task %s export' % self.filter
+        else:
+            task_command = 'task export'
+        task_command = 'task %s export' % filter
```

Comment:
`filter` / `self.filter`. We want to remove this line anyway.
```
Command `task <class 'filter'> export` returned non-zero exit status 2 (CommandError) py3.py line 929.
Traceback
CommandError: Command `task <class 'filter'> export` returned non-zero exit status 2
  File "/src/py3status/py3status/module.py", line 729, in run
    response = method()
  File "taskwarrior.py", line 49, in taskWarrior
    task_json = json.loads(self.py3.command_output(task_command))
  File "/src/py3status/py3status/py3.py", line 929, in command_output
    msg, error_code=retcode, error=error, output=output
```

Reply:
@tobes I'd remove `start.before:today` in `filter` to make `taskwarrior` more usable. Proceed?

Reply:
maybe we should just keep the original filter (I don't use this module so no expert)

The line does need removing but then I think it'd be good to merge and close this if we can

Reply:
From what I gather... It's an author's `filter`. Any tasks we add recently won't show up until the next day... which would confuse those who wanted to try `taskwarrior` module but couldn't get it to work with `taskw`... or annoying for those who understand `taskw`well... because this is hardcoded.

`status:pending` can stay as we don't want completed tasks in the bar. Use `task 1 done` to mark it completed. Use `task 1 delete` to delete the task (Users shouldn't delete their regularly if they want charts or such).

Reply:
@lasers I'm happy for you to take the lead on this as you have better knowledge on this.  Just waiting for the bug fix now

Reply:
After making a full-fledged `taskwarrior` module https://github.com/ultrabug/py3status/pull/1107 from scratch, I like to think I understand a thing or two about `taskwarrior`. 👍 @regnat Maybe you could test it for fun before we close it?